common/script/index.coffee

```diff
@@ -1740,14 +1744,22 @@ api.wrap = (user, main=true) ->
           dailyChecked += 1
         else
           # dailys repeat, so need to calculate how many they've missed according to their own schedule
+          console.log("task: " + task.text) # XXX Alys to remove these
           scheduleMisses = 0
+          keepGoing = true # XXX Alys to improve how we break out of _.times
           _.times daysMissed, (n) ->
```

Comment:
And just the day after I was telling my housemate how useful underscore.js is. :-/

Thanks! :)